randovania/games/fusion/logic_database/Sector 6 NOC.txt

```diff
@@ -709,7 +715,15 @@ Extra - room_id: [18]
   > Pickup (Hidden Power Bomb Tank)
       All of the following:
           Can Use Any Bombs
-          Movement (Beginner) or Can Use Springball
+          Any of the following:
+              All of the following:
+                  # Vanilla Method - freeze a spikey on the crumble blocks (and have regular bombs only)
```

Comment:
capitalization. also i think the enemy's name is `Owtch` not spikey. Could have `Owtch (Spikey)` maybe for clarity.

Reply:
Fixed - I read the name off the discord message and had just rolled with it 😁 
It's the only enemy type in the room, so the descriptor seems unnecessary